config/locales/en.yml

```diff
@@ -36,6 +36,8 @@ en:
         create: "Add"
   activerecord:
     attributes:
+      gsa18f/procurement:
+        recurring: "Recurring charge"
```

Comment:
so we are using I18n! Interesting. How strict are we being about using it? Seems like there is quite a bit of text in the view templates...